### DISCUSSION TITLE
Commit toolbar button rollbacks changes

### DISCUSSION THREAD

**Discussion by kAlvaro** [dbeaver/dbeaver#35630] [Q&A] [answered]
I'm using version 24.2.0.202409011551 on Ubuntu 22.04 LTS (snap release) with PostgreSQL/14. I have a custom connection type for production databases, where I always to want commit changes manually.

![image](https://github.com/user-attachments/assets/b2fbabdc-0294-4fb2-9715-1efc16d23d89)

It's been working fine for months, but I've just discovered that now all my pending inserts get rollbacked as soon as I hit the "Commit" toolbar button. I make changes by running `INSERT INTO` statements in a SQL tab.

![image](https://github.com/user-attachments/assets/b51760cb-b9d9-4064-826f-ddc82edd2a4b)

I need to explicitly type `COMMIT;` in my SQL tab and run it.

It might have been this way for some weeks, but I always thought I just forgot to commit.

Is it a known bug? Is there some obscure setting that got corrupted?

**Comment by E1izabeth** [comment]
Please report it as a bug report. We haven't heard about that previously. Also please provide clear steps to reproduce your issue in bug report.

**Comment by kAlvaro** [comment, marked as answer]
@E1izabeth I tried to apply Production settings to my local test account and I couldn't reproduce it. Then I realised the program had auto-updated to 24.2.1.202409221725, and the actual production connection no longer exhibits this behaviour. So, whatever the issue was, it's gone. 🤷
